Source/Lib/Codec/EbEncHandle.c

```diff
@@ -1724,7 +1724,9 @@ void LoadDefaultBufferConfigurationSettings(
     }
     threadUnit = totalThreadCount / EB_THREAD_COUNT_MIN_CORE;
 
-    sequenceControlSetPtr->inputOutputBufferFifoInitCount = inputPic + SCD_LAD;
+    sequenceControlSetPtr->inputOutputBufferFifoInitCount = inputPic + SCD_LAD +
+        sequenceControlSetPtr->staticConfig.lookAheadDistance +
+        sequenceControlSetPtr->staticConfig.intraPeriodLength;
```

Comment:
is lookAheadDistance alone good enough?

Reply:
Unfortunately, either lookAheadDistance or intraPeriodLength alone isn't good enough...